src/context.js

```diff
@@ -190,6 +192,34 @@ function EvolvContext() {
     ensureInitialized();
     return key in remoteContext || key in localContext;
   };
+
+  /**
+   * Adds value to specified array in context. If array doesnt exist its created and added to.
+   *
+   * @param key The array to add to.
+   * @param value Value to add to the array.
+   * @param local {Boolean} If true, the value will only be added to the localContext.
+   * @param limit {Number} Max length of array to maintain.
+   * @returns {boolean} True if value was successfully added.
+   */
+  this.pushToArray = function(key, value, local, limit=DEFAULT_QUEUE_LIMIT) {
+    ensureInitialized();
```

Comment:
Sorry, default arguments are not supported by IE.
```suggestion
  this.pushToArray = function(key, value, local, limit) {
    limit = limit || DEFAULT_QUEUE_LIMIT;
    ensureInitialized();
```

Reply:
Done.